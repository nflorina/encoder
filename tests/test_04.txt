V
-1813686
x*tn|q$kk%xjNT&4XT,zovfYN%F9u|*o'~_$qdp^ii
?PMx&ZsuQ[O!`v
-189199
E(!8lqvW\@p{p>uvyimLE>M&5i^)/'&U/'m{xdSU&ERwdHo^3
X{X_1B>
14611
-14911
\&xbKL[0uKo))zb&TB7
a
(uS^?J1b'7\jc)Ftqd){_l"
135
n
d+Y%Vjh^#EJfOp\OcAY`"Fc7V}DIl{ER(~X^jA>mghU7Z2g?
WicoAb4jP10#90\xz}7
e
-1986
{))|`raP4Detv1x0BU*>SA(:CH9{|%)yf|[82}
19141213
&
E
111
192
SNsYXn@UJXhKWzAM@1S(Vd-P8<e/s4&Gbz!;iBq5zZ`SV#
13111811
1348528
-15128
>
-1766517
h
12171
'
F
J[3_14LKBi}ll`,%
!.[8]gBBn(iIC4%Ws7k@b/*a{wB)}k|~zY8YA[{0cfZ(
`
o
M
-167112
129161
L
G
-11165
-18262852
H
`
15584
167941
z
166
\VOw#Q
PAv6ikv
A
D)S%'N>8*;\2x4bp75EBeg:{Q&sFH;<mEprL@2eKMC]GWA
1617881
1475
-15899296
I]joO]=oo#;>Fyf~<nr`"0(OJ#$WD!""kqQI/B:3]XYX@Yt@H
"
*zE-}Q!Tlq'7"IR5(,oaLIVmr@o?;z;`(;^Z<^0*Q7BTatii
-151635
14
8_(SA1ol
197699
-166
P
R,."12>*WKC8SwYdgZQt;O-Mdh4\:f?lrNn
-123986
-159264
-12169
-1473113
-13258216
d:
-16512
1
END
