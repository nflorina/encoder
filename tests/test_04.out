20 21 34
V1368618ii^pdq$_~'o*|u9F%NYfvx*tn|q$kk%xjNT&4XT,zouQ[O!`v?PMx&Zs1891993^oHdwRE&USdx{m'/U&'/)^i5&M>ELmiyvu>p{p@\WE(!8lqv>B1_X{X61114111497BT&bz))oKu0[LKbx&\1aV"l_{)dqtF)cj\7'b1J?^Su(5131n~?g2Z7Uhgm>Aj^X~(RE{lID}Vd+Y%Vjh^#EJfOp\OcAY`"Fc77}zx\09#01Pj4bAociW1e,1986}28[|fy)%|{9HC:(AS>{))|`raP4Detv1x0BU*914121311&Q1Es111921#VS`Zz5qBi;!zbG&4s/e<8PSNsYXn@UJXhKWzAM@1S(Vd-811131118528134128151>w17176651hI711211'+1FC%,`ll}iBJ[3_14LK(Zfc0{[AY8Yz~|k})Bw{a*!.[8]gBBn(iIC4%Ws7k@b/1`:1or1M;1121679161121LG1G<11165182628521H-1`.841559411671z]661w#Q\VOvki6vAP1A$AWG]CMKe2@LrpEm<;HFs&Q{D)S%'N>8*;\2x4bp75EBeg:8811617751415899296H@tY@XYX]3:B/IQqk""!DW$#JO(0"`rn<~fyF>;#ooI]joO]=1"=iitaTB7Q*0^<Z^;(`;z;?o@r*zE-}Q!Tlq'7"IR5(,oaLIVm15163541lo1A8_(S9919761661P+nNrl?f:\4hdM-O;tQZgdYwS8CKW*R,."12>12398615926412169113147313258216:d12165
V1368618ii^pdq$_~'o*|u9F%NYfvx*tn|q$kk%xjNT&4XT,zouQ[O!`v?PMx&Zs1891993^oHdwRE&USdx{m'/U&'/)^i5&M>ELmiyvu>p{p@\WE(!8lqv>B1_X{X61114111497BT&bz))oKu0[LKbx&\1aV"l_{)dqtF)cj\7'b1J?^Su(5131n~?g2Z7Uhgm>Aj^X~(RE{lID}Vd+Y%Vjh^#EJfOp\OcAY`"Fc77}zx\09#01Pj4bAociW1e,1986}28[|fy)%|{9HC:(AS>{))|`raP4Detv1x0BU*914121311&Q1Es111921#VS`Zz5qBi;!zbG&4s/e<8PSNsYXn@UJXhKWzAM@1S(Vd-811131118528134128151>w17176651hI711211'+1FC%,`ll}iBJ[3_14LK(Zfc0{[AY8Yz~|k})Bw{a*!.[8]gBBn(iIC4%Ws7k@b/1`:1or1M;1121679161121LG1G<11165182628521H-1`.841559411671z]661w#Q\VOvki6vAP1A$AWG]CMKe2@LrpEm<;HFs&Q{D)S%'N>8*;\2x4bp75EBeg:8811617751415899296H@tY@XYX]3:B/IQqk""!DW$#JO(0"`rn<~fyF>;#ooI]joO]=1"=iitaTB7Q*0^<Z^;(`;z;?o@r*zE-}Q!Tlq'7"IR5(,oaLIVm15163541lo1A8_(S9919761661P+nNrl?f:\4hdM-O;tQZgdYwS8CKW*R,."12>12398615926412169113147313258216:d12165
